Source/SmallBasic.Editor/Libraries/GraphicsWindowLibrary.cs

```diff
@@ -234,6 +234,8 @@ public void Set_FontName(string value)
 
         public void Set_FontSize(decimal value) => this.libraries.Styles = this.libraries.Styles.With(fontSize: value);
 
+        public Task Set_Height(decimal value) => JSInterop.Layout.ShowMessage("Setting GraphicsWindow.Height is not supported in this version of SmallBasic", string.Empty);

```

Comment:
>ShowMessage [](start = 66, length = 11)

Note that ShowMessage pops up as a modal dialog in Chrome.  If this is too disruptive, I can try to find another way.